### PR TITLE
OOD-12-13-40: no sentryDns or winstonGelf for service-provider Fd

### PR DESCRIPTION
**Pull request by rundtjan**
Tickets:
https://funidata.atlassian.net/browse/OOD-12
https://funidata.atlassian.net/browse/OOD-13
https://funidata.atlassian.net/browse/OOD-40

### CODE REVIEW

**Comment by rundtjan** [comment]
Awaiting review by Funidata before merging.

**Comment by rundtjan** [comment]
Some changes in the pipeline, no merge still.

**Review by valtterikantanen — APPROVED** [review]

**Pull request merged**
